test/integration.js

```diff
@@ -226,23 +292,35 @@ tape('POST /channel/{id}/{events,validator-messages}: wrong authentication', fun
 })
 
 tape('POST /channel/{id}/{validator-messages}: wrong signature', function(t) {
-	const stateRoot = "6def5a300acb6fcaa0dab3a41e9d6457b5147a641e641380f8cc4bf5308b16fe"
+	let stateRoot = ""
```

Comment:
do you have any clue why this was not failing considering we had the bug in `follower`